mantidproject/mantid
# Fit function docs addition

**Description of work.**

**Report to:** James Lord

**To test:**

Build the html docs and check that the section **Errors** in `concepts/FitFunctionsInPython.html` makes sense.



Fixes fourum post http://forum.mantidproject.org/t/error-values-from-a-fit/457/2.
*There is no associated issue.*



*This does not require release notes* because it is documentation.


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
